install.ps1

```diff
@@ -1,84 +1,86 @@
 # Enable TLS 1.2 since it is required for connections to GitHub
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$ErrorActionPreference = "Stop"
 
-Write-Host "Beginning installation of spicetify-bloom"
-Write-Host "https://github.com/nimsandu/spicetify-bloom"
+Write-Host -Object "Beginning installation of spicetify-bloom"
+Write-Host -Object "https://github.com/nimsandu/spicetify-bloom"
 
 # Give time for user to cancel via CTRL+C
-Start-Sleep -s 3
+Start-Sleep -Seconds 3
 
-$checkSpice = Get-Command spicetify -ErrorAction Silent
-if ($null -eq $checkSpice) {
-  Write-Host -ForegroundColor Red "Spicetify not found. Installing that for you..."
-  Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" | Invoke-Expression
+if ( -not (Get-Command -Name spicetify -ErrorAction SilentlyContinue) ) {
+  Write-Host -Object "Spicetify not found. Installing it for you..." -ForegroundColor Red
+  Invoke-WebRequest -Uri "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" -UseBasicParsing | Invoke-Expression

```

Comment:
Even though PowerShell documentation does utilize `-Uri` parameter, it can be omitted. There's no need to change it.

Reply:
same link